russh-config/src/lib.rs

```diff
@@ -166,22 +171,10 @@ pub fn parse(file: &str, host: &str) -> Result<Config, Error> {
                         debug!("{:?}", key);
                     }
                 }
-            } else if lower.as_str() == "host"
-                && value
-                    .split_whitespace()
-                    .any(|x| check_host_against_glob_pattern(host, x))
-            {
-                let mut c = Config::default(host);
-                c.port = 22;
-                config = Some(c)
             }
         }
     }
-    if let Some(config) = config {
-        Ok(config)
-    } else {
-        Err(Error::HostNotFound)
-    }
+    Ok(config)
```

Comment:
With glob directives, it makes little sense to return HostNotFound error, I may return this error, but I think most of the time users just want to parse all relevant host settings, and will `unwrap_or_default` anyways after config parse call.

Reply:
Makes sense